.storybook/preview.js

```diff
@@ -1,8 +1,12 @@
 import "@storybook/addon-actions/register";
-import "@components/setupTwind";
```

Comment:
This file doesn't really do anything except the setup so it felt quite shallow. Removed the file and did the twind setup below.